static/Makefile

```diff
@@ -62,7 +62,8 @@ cross-win: cross-win-engine cross-win-plugins
 	cd $(CLI_DIR) && VERSION=$(GEN_STATIC_VER) docker buildx bake --set binary.platform=windows/amd64 binary
 	mkdir -p build/win/amd64/docker
 	cp $(CLI_DIR)/build/docker-windows-amd64.exe build/win/amd64/docker/docker.exe
-	cp $(ENGINE_DIR)/bundles/cross/windows/amd64/dockerd-$(GEN_STATIC_VER).exe build/win/amd64/docker/dockerd.exe
+	cp $(ENGINE_DIR)/bundles/cross/windows/amd64-daemon/dockerd-$(GEN_STATIC_VER).exe build/win/amd64/docker/dockerd.exe
+	cp $(ENGINE_DIR)/bundles/cross/windows/amd64-daemon/docker-proxy-$(GEN_STATIC_VER).exe build/win/amd64/docker/dockerd.exe
```

Comment:
This PR fixes the issue, but after this I want to look in the upstream moby repository "why" this path changed, because there's also some inconsistency here (`<arch>-daemon` for some binaries, and `<arch>` for others); see https://github.com/docker/docker-ce-packaging/pull/554/commits/09541b553c2f7646626efcc2797bd92ee48a66d0